src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -168,10 +168,21 @@ Just set the `pinned` prop.
 				:force-menu="forceMenu"
 				:default-icon="menuIcon"
 				@update:open="onMenuToggle">
-				<ActionButton v-if="editable && !editingActive" icon="icon-rename" @click="handleEdit">
+				<template #icon>
+					<!-- @slot Slot for the custom menu icon -->
+					<slot name="menu-icon" />
+				</template>
+				<ActionButton v-if="editable && !editingActive" @click="handleEdit">
+					<template #icon>
+						<Pencil :size="24" decorative />
```

Comment:
```suggestion
						<Pencil :size="20" decorative />
```
🙏 🙏

Reply:
@marcoambrosini I would suggest to not mix different icon sizes for now and adjust the size library wide in a single PR once it is decided. Hence, keeping `24` in this PR and I promise to create a PR to adjust it in case 😉 Would this be ok?

Reply:
Sounds good!